integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSession.scala

```diff
@@ -248,7 +251,7 @@ object CarbonSession {
 
         session = new CarbonSession(sparkContext, None, !enableInMemCatlog)
         val carbonProperties = CarbonProperties.getInstance()
-        if (storePath != null) {
+        if (storePath != null && StringUtils.isNotBlank(storePath)) {
```

Comment:
remove 'storePath != null', StringUtils.isNotBlank already includes this judgement.